PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/startup/WACPStartUp.java

```diff
@@ -147,6 +149,8 @@ private void initialize() {
 	        }
 			EventUtil.logEvent(EventType.info, SYSTEM, PLUGIN_LOAD, null, null);
 	        AppInsightsClient.createByType(AppInsightsClient.EventType.Plugin, "", AppInsightsConstants.Load, null, true);
+	        
+	        new CustomerSurveyHelper().showFeedbackNotification();
```

Comment:
Use a singleton here?

Reply:
will finally use singleton after refactoring with intellj code

Reply:
There used to be an issue for IntelliJ that if users open multi-IntelliJ window, intellij will invoke `initialize()` for each window, and may pop up survey windows multi-times, so change to use singleton and check whether has pop up before, i'm not sure whether Eclipse have the same issue.

Reply:
Because eclipse.exe are running in different process, it is hard/not necessary to detect multiple instances, pop up two times(when opening more than one eclipse instances) will be ok